grai-integrations/source-dbt/src/grai_source_dbt/adapters/adapters.py

```diff
@@ -116,6 +115,11 @@ def build_grai_metadata_from_edge(current: Edge, version: Literal["v1"] = "v1")
         "version": version,
         "edge_type": current.edge_type.value,
         "tags": [config.metadata_id],
+        "code": Code(
+            code=current.definition,
+            language={"language_name": "SQL", "file_extension": ".sql"},
```

Comment:
I was hoping to use the `SQL` class from `grai_schemas.v1.generics`, however I seem to get an error.